File: human_class/Human.java
// javac -d . Human.java;
// java jp.co.firstclass.Human;
// package jp.co.firstclass;
import java.util.*;

public class Human {
    private String name;
    private String sex;

    public Human(Map<String, String> opt){
        this.name = opt.get("name");
        this.sex = opt.get("sex");
    }

    public void sayName() {
        System.out.println("My name is " + this.name);
    }

    public void saySex() {
        System.out.println("My sex is " + this.sex);
    }

    public static void main(String args[]) {
        Map<String, String> opt = new HashMap<String, String>();
        opt.put("name", "FirstName LastName");
        opt.put("sex", "male");
        Human human = new Human(opt);
        human.sayName();
        human.saySex();
    }
}

File: human_class/Human.py
class Human:
	def __init__(self, opt):
		self.name = opt["name"]
		self.sex = opt["sex"]

	def sayName(self):
		print("My name is " + self.name)

	def saySex(self):
		print("My sex is " + self.sex)

if __name__ == "__main__":
	person1 = Human({"name":"FirstName LastName", "sex":"male"})
	person1.sayName()
	person1.saySex()

